scss/window/_layout.scss

```diff
@@ -21,19 +21,23 @@
 
     // Title bar
     .k-window-titlebar {
-        padding: $window-titlebar-padding-y $window-titlebar-padding-x;
-        min-height: 16px;
-        border-width: 0 0 1px;
-        border-style: solid;
+        padding: $window-titlebar-padding;
+        min-height: $window-titlebar-min-height;
+        border-width: $window-titlebar-border-width;
+        border-style: $window-titlebar-border-style;
         white-space: nowrap;
         display: flex;
         flex-direction: row;
         flex-shrink: 0;
         align-items: center;
     }
+
+    // Title bar title text
     .k-window-title {
         margin: 0;
-        font-size: $font-size-lg;
+        font-size: $window-title-font-size;
+        font-family: $window-title-font-family;
+        text-transform: $window-title-text-transform;
```

Comment:
I think the variables are overly specific and should not be exposed in that format. I understand your requirements for the headers to be consistent, and perhaps we can introduce another way to achieve this, through another issue / PR. An idea that comes to mind is to expose a `@mixin` that can be redefined in consuming code, and is used across all component headers.